.github/workflows/test.yml

```diff
@@ -76,3 +87,41 @@ jobs:
         if: "${{ env.PREVENT_PERCY_TESTS != 'true' }}"
         run: |
           npx @percy/cli snapshot docs/.vitepress/dist/ --exclude "**/404.html"
+
+      - name: Publish Previews
```

Comment:
suggestion: can we move these actions above the Percy block? 

Sometimes Percy hangs/stalls or we run out of free credits

Reply:
sure, changed.